03_gamedesign/index.tex

```diff
@@ -1 +1,139 @@
+
+\begin{table}[h]
+    \centering
+    \caption{Scores given for each stack based on requirements}
+    \label{table:techstackscores}
+    \begin{tabular}{|c|c|c|c|c|c|c|}
+        \hline
+        Stack                     &
+        \makecell{Easy              \\ to \\ learn}              &
+        \makecell{Easy              \\ to \\ setup}              &
+        \makecell{Cross-platform} &
+        \makecell{Easily            \\ maintainable}             &
+        \makecell{Friendly          \\ features}                 &
+        \makecell{Final             \\ score \\ (out of 50)}
+        \\
+        \hline
+        Multi-language            &
+        6                         &
+        2                         &
+        0                         &
+        2                         &
+        10                        &
+        20
+        \\
+        \hline
+        Python                    &
+        8                         &
+        6                         &
+        8                         &
+        3                         &
+        8                         &
+        34
+        \\
+        \hline
+        C\#                       &
+        6                         &
+        5                         &
+        8                         &
+        9                         &
+        8                         &
+        37
+        \\
+        \hline
+        Go                        &
+        8                         &
+        8                         &
+        10                        &
+        8                         &
+        6                         &
+        40
+        \\
+        \hline
+    \end{tabular}
+\end{table}
+
+
+\subsection{Visualisations}
+\label{sec:GD:implementation:visualisations}
+
+\subsubsection{Toolchain}
+A benefit from the choice of the Go technology stack was that it supports compiling source code into WebAssembly out of the box. WebAssembly can be run efficiently in most modern browsers, which meant that in-browser simulations can be run and then visualised on a website.
+
+The website (\url{https://somas2020.github.io/SOMAS2020/}) features... % TODO:- Vis team: yp717 et al.
+
+\subsection{Engineering Practices}
+\label{sec:GD:implementation:practices}
+
+Developing and maintaining a codebase with contributions from around 40 developers was projected to be not easy. Therefore, good software engineering practices and rules were employed to be upheld by all contributors to make the process smoother and--where possible--automated.

```

Comment:
to be not easy -> not to be easy

Reply:
changed to `projected to be non-trivial` -- cleaerer